specs-go/config.go

```diff
@@ -495,7 +495,7 @@ type WindowsNetwork struct {
 // WindowsHyperV contains information for configuring a container to run with Hyper-V isolation.
 type WindowsHyperV struct {
 	// UtilityVMPath is an optional path to the image used for the Utility VM.
-	UtilityVMPath string `json:"utilityvmpath,omitempty"`
+	UtilityVMPath string `json:"utilityVMPath,omitempty"`
```

Comment:
hey @wking and @jhowardmsft.  i'll be taking over responsibility for the solaris containers work from @anuthan.  i did an audit of our use of camel case and i'd prefer not to change it.  in general we've tried to have our property names map to zonecfg.1m properties.  "defrouter" being a good example.  the places where we have used camel case is for zonecfg properties that have hyphens in them.  so for example the zonecfg "allowed-address" property has been mapped to allowedAddress.  hopefully this seems sensible to folks.

that said, john correctly noted that some of our doc links are broken so i filed issue #864 on that.